cmd/test/examples/python/test-data/python/main.py

```
@@ -0,0 +1,21 @@
+"""Sample sources for python provider tests (python-sample-rule-001–004)."""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)